tmt/steps/prepare/artifact/providers/repository.py

```diff
@@ -165,7 +189,7 @@ def parse_rpm_string(pkg_string: str) -> dict[str, str]:
 
     :param pkg_string: The package string, e.g., "docker-ce-1:20.10.7-3.el8.x86_64".
     :raises ValueError: if the package string is malformed.
-    :return: A dictionary of RPM components.
+    :returns: A dictionary of RPM components.
```

Comment:
Unrelated change. Both `return` and `returns` are valid afaik, but do let me know if I am off on that.